### DISCUSSION TITLE
Is it possible to map ESRI geometry?

### DISCUSSION THREAD

**Discussion by fujiokae** [dbeaver/dbeaver#13669] [Q&A] [answered]
I manage an ESRI enterprise geodatabase (EGDB) with PostgreSQL and wonder if I can map the geometry using the Geometry Viewer?

I also manage PostGIS databases and when I choose a geometry column, the Geometry Viewer automatically shows up. This does not happen with the EGDB. I could run a query like 
SELECT st_astext(shape) as geom_text, shape FROM ersi_feature_class;
The text representation of the shape is like "POINT ( -173.50000000 52.00000000)" but of course it's a text, so the map does not show up either.

Is there a way to make the Geometry Viewer available for the ESRI geometry? Any plugin or library?

Thanks for your help!

DBeaver Community 21.1.4

**Comment by ShadelessFox** [comment, marked as answer]
Hello @fujiokae,

ERSI geometry is not supported. You can create a [feature request](https://github.com/dbeaver/dbeaver/issues/new?assignees=&labels=feature+request%2C+wait+for+review&template=feature_request.md&title=), so the issue can be tracked.

**Reply by fujiokae** [reply]
Thank you for a quick response.
I have submitted a request https://github.com/dbeaver/dbeaver/issues/13692#issue-980390897.
